tests/fgpyo/util/test_metric.py

```diff
@@ -613,11 +613,30 @@ def test_writer_from_str(tmp_path: Path) -> None:
 
     with fpath.open("r") as f:
         assert next(f) == "foo\tbar\n"
+        assert repr(f.newlines) == repr("\r\n")
         assert next(f) == "abc\t1\n"
         with pytest.raises(StopIteration):
             next(f)
 
 
+def test_writer_with_kwargs(tmp_path: Path) -> None:
```

Comment:
Add this and fix up the test:
```suggestion
@pytest.mark.parameterize("lineterminator", ["\n", "\r", "\r\n"])
def test_writer_with_kwargs(tmp_path: Path, lineterminator: str) -> None:
```